bot/seasons/evergreen/game.py

```diff
@@ -0,0 +1,323 @@
+
+        Also support amount parameter, what max is 25 and min 1, default 5. Use quotes ("") for genres with multiple
+        words.
+        """
+        # Capitalize genre for check
+        genre = genre.capitalize()
+
+        # Check for amounts, max is 25 and min 1
+        if amount > 25:
+            await ctx.send("You can't get more than 25 games at once.")
+            return
+        elif amount < 1:
+            await ctx.send("You can't get less than 1 game.")
+            return
+
+        # Get games listing, if genre don't exist, show help.
+        try:
+            games = await self.get_games_list(self.http_session, amount, self.genres[genre],
+                                              offset=random.randint(0, 150))
+        except KeyError:
+            await ctx.send_help('games')
+            return
+
+        # Create pages and paginate
+        pages = [await self.create_page(game) for game in games]
+
+        await ImagePaginator.paginate(pages, ctx, Embed(title=f'Random {genre} Games'))
+
+    @games.command(name='top', aliases=['t'])
+    async def top(self, ctx: Context, amount: int = 10) -> None:
+        """
+        Get current Top games in IGDB.
+
+        Support amount parameter. Max is 25, min is 1.
+        """
+        if amount > 25:
+            await ctx.send("You can't get more than top 25 games.")
+            return
+        elif amount < 1:
+            await ctx.send("You can't get less than 1 top game.")
+            return
+
+        games = await self.get_games_list(self.http_session, amount, sort='total_rating desc',
+                                          additional_body="where total_rating >= 90; sort total_rating_count desc;")
+
+        pages = await self.get_pages(games)
+        await ImagePaginator.paginate(pages, ctx, Embed(title=f'Top {amount} Games'))
+
+    @games.command(name='genres', aliases=['genre', 'g'])
+    async def genres(self, ctx: Context) -> None:
+        """Get all available genres."""
+        await ctx.send(f"Currently available genres: {', '.join(f'`{genre}`' for genre in self.genres)}")
+
+    @games.command(name='search', aliases=['s'])
+    async def search(self, ctx: Context, *, search: str) -> None:
+        """Find games by name."""
+        lines = await self.search_games(self.http_session, search)
+
+        await LinePaginator.paginate((line for line in lines), ctx, Embed(title=f'Game Search Results: {search}'))
+
+    @games.command(name='company', aliases=['companies'])
+    async def company(self, ctx: Context, amount: int = 5) -> None:
+        """
+        Get random Game Companies companies from IGDB API.
+
+        Support amount parameter. Max is 25, min is 1.
+        """
+        if amount > 25:
+            await ctx.send("You can't get more than 25 companies at once.")
+            return
+        elif amount < 1:
+            await ctx.send("You can't get less than 1 company.")
+            return
```

Comment:
I think just checking if `1 <= amount <= 25` would be sufficient. You can then explain to the user that `amount` escapes the allowed range, and show what the range is.